### DISCUSSION TITLE
Is the tex file compiled locally or on the overleaf servers

### DISCUSSION THREAD

**Discussion by lyyc199586** [overleaf-workshop/Overleaf-Workshop#209] [Q&A]
Hi, all

This seems to be a great ext! I am just curious with this ext, is tex file still compiled on the overleaf servers or locally?

**Comment by iamhyc** [comment]
The overleaf project is hosted on the server and therefore compiled on the server.
The compiled results are then fetched and displayed in VS Code.

If you need to compile the project locally (e.g., via "LaTeX-Workshop"), you can try open the project locally: 
https://github.com/iamhyc/Overleaf-Workshop/blob/master/docs/wiki.md#open-project-locally

**Reply by lyyc199586** [reply]
I see, so you can do both, great
